### DISCUSSION TITLE
apt install error

### DISCUSSION THREAD

**Discussion by bad-friday** [Motion-Project/motion#1731] [General]
Get the following error when sudo apt install  ./pi_bookworm_motion_4.6.0-1_arm64.deb

N: Download is performed unsandboxed as root as file '/home/swe/pi_bookworm_motion_4.6.0-1_arm64.deb' couldn't be accessed by user '_apt'. - pkgAcquire::Run (13: Permission denied)

**Comment by bmn-m** [comment]
This is not a motion specific message and the installation should work as intended. 
Even when apt runs as root, for security reasons it tries to download packages as user _apt. In your case apt tries to (down)load the package from your home directory, but it has no sufficient rights. So apt is accessing the .deb file as root and informs you about that.
